central/detection/lifecycle/manager.go

```diff
@@ -20,8 +20,9 @@ import (
 )
 
 const (
-	rateLimitDuration            = 10 * time.Second
-	indicatorFlushTickerDuration = 1 * time.Minute
+	rateLimitDuration             = 10 * time.Second
+	indicatorFlushTickerDuration  = 1 * time.Minute
+	deploymentFlushTickerDuration = 1 * time.Minute
```

Comment:
Can turn up the check on this because a single go routine checking a queue is cheap and most of the time no work will occur

Reply:
5 - 10s? This will allow the tests to run faster too :)